server/dummy-data.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "name": "Jamie's Bakery",
+    "address": "123 Fake Street",
+    "howToOrder": "Online",
+    "website": "https://jamies-bakery.com/",
+    "openingHours": "Online between 8am - 8pm online for next day delivery at the latest",
+    "notes": "Still delivering fresh made bread and delicious treats every day",
+    "socialMedia": "@jamiesbakery",
+    "tags": [
+      "bread", "delivery"
+    ],
```

Comment:
Also need to add in 
"category": "food"

Reply:
No probs 👍Is it possible to have multiple categories? Like a book shop that does coffee?

Reply:
Argh - this is the whole problem that we have to deal with.  Becky and I were having this discussion on the google doc...  Does that become too complicated? At the moment the categories are Community, Fitness, Food, Shops, Services, Wine and Beer.  Is it best to design for possible multiple categories in case we change it down the line?

Reply:
Yeah I think so, even if initially we just have one category per business - we can still store it in an array so we'd be ready to support multiple categories